paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -110,9 +112,22 @@ data class DeviceConfig(
         versionQualifier = VersionQualifier()
       }
 
+  private val currentWidth: Int
+    get() = if (orientation == ScreenOrientation.PORTRAIT) {
```

Comment:
Should this be exhaustive considering [square](https://developer.android.com/reference/android/content/res/Configuration#ORIENTATION_SQUARE) is unsupported?

```
Portrait - min
Landscape - max
Square - error
```

Reply:
Makes sense to me :+1:

Reply:
nah, Square is deprecated since API 16

Reply:
whoops, nvm.  Brian beat me to it.  No biggy.

Reply:
There's a couple other screen orientations so I'll probably update the error to say portrait and landscape are the only supported ones. I want to look into the others at some point to see if they're worth supporting

Reply:
They might be trivial to support, right?